docs/Systems/Tailwind/components/CardContent.tsx

```diff
@@ -4,21 +4,23 @@ import { Heading } from "@kaizen/typography"
 
 type CardContentProps = {
   title: string
-  Illustration: React.ReactElement
+  illustration: React.ReactElement
 }
 
 export const CardContent = ({
   title,
-  Illustration,
+  illustration,
 }: CardContentProps): React.ReactElement => (
   <div className="flex w-[400px] flex-col items-center p-24 pt-48">
     <Heading variant="heading-3">{title}</Heading>
     <p className="pt-12 pb-24 text-center font-family-paragraph text-heading-6 font-weight-paragraph text-purple-800">
       Lorem ipsum oogum boogum. Tutant meenage neetle teetles.
     </p>
-    <div className="w-[200px]">{Illustration}</div>
+    <div className="w-[200px]">{illustration}</div>
```

Comment:
Lowercase for a react component? Shouldn't this be `<Illustration />`?

Reply:
It's a prop though 🤷🏻